### PR TITLE
Add contributors section

### PR DESCRIPTION
**Pull request by alkozp**
Hi team, check the contributors section please

### CODE REVIEW

File: js/app/app.js [hunk 1]
```diff
@@ -17,5 +17,60 @@ document.addEventListener("DOMContentLoaded", () => {
...
+/** ### async func get list contributors for repo from github */
+const getContributors = async (repoName) => {  
+    let request = await fetch(`https://api.github.com/repos/${repoName}/contributors`, {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json',
+        }
+    });
+
+    let contributorsList = await request.json();
```
Comment (kostasx) [inline js/app/app.js:31]: Use `const` when the value does not change.

File: js/app/app.js [hunk 2]
```diff
@@ -17,5 +17,60 @@ document.addEventListener("DOMContentLoaded", () => {
 })
 
 
+//** contributors section */
+
+/** ### async func get list contributors for repo from github */
+const getContributors = async (repoName) => {  
+    let request = await fetch(`https://api.github.com/repos/${repoName}/contributors`, {
```
Comment (kostasx) [inline js/app/app.js:24]: Wrap the `await` sections in a `try/catch` to handle any errors.

File: css/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    margin: 0px;
+    padding: 0px;
+}
+
+#contributors img {
+    border-radius: 50%;
+    height: 10rem;
+    width: 10rem;
+    box-shadow: 0 4px 8px 0 var(--contributors-box-shadow-color), 0 6px 20px 0 var(--contributors-box-shadow-color);
+    /* shape-outside: circle();
```
Comment (kostasx) [inline css/styles/style.css:76]: Perhaps, we can remove these comments when we're done and we are ready with the final version.

**Review by kostasx — COMMENTED** [review]
Great work!

Left some comments.

File: js/app/app.js (hunk 1 above)
Comment (alkozp) [reply js/app/app.js]: Done

File: js/app/app.js (hunk 2 above)
Comment (alkozp) [reply js/app/app.js]: Done

File: css/styles/style.css (hunk 3 above)
Comment (alkozp) [reply css/styles/style.css]: Done

**Comment by kostasx** [comment]
LGTM!

File: css/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    max-width: 900px;
+    margin: 0 auto;
+    padding: 3rem;
+}
```
Comment (MaryRichelle) [inline css/styles/style.css:6]: i think its better if we remove max-width from body element and maybe padding also. you can achieve same task with wrapping your element with div tag and add all that styling there

File: css/styles/style.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+/* contributors section styles*/
+
+@media screen and (max-width: 400px) {
```
Comment (MaryRichelle) [inline css/styles/style.css:10]: It  best practice is to add media queries at the end of the CSS file. and i suggest implementing best practices if it is possible Alex.

**Review by MaryRichelle — CHANGES_REQUESTED** [review]
Over all great work Alex. Thanks for your pull request.

**Pull request merged**